python_modules/dagster-pipes/dagster_pipes/version.py

```diff
@@ -1 +1 @@
-__version__ = "1!0+dev"
+__version__ = "1.5.6"
```

Comment:
[1] this will need to be reverted back to "1!0+dev"

Reply:
sorry, only did that so I could test locally. I'll revert it in a minute.